integration/apps/wrf/README.md

```diff
@@ -0,0 +1,29 @@
+WRF was developed at the National Center for Atmospheric Research
+(NCAR) which is operated by the University Corporation for
+Atmospheric Research (UCAR). NCAR and UCAR make no proprietary
+claims, either statutory or otherwise, to this version and release
+of WRF and consider WRF to be in the public domain for use by any
+person or entity for any purpose without any fee or charge. UCAR
+requests that any WRF user include this notice on any partial or
+full copies of WRF. WRF is provided on an "AS IS" basis and any
+warranties, either express or implied, including but not limited
+to implied warranties of non-infringement, originality,
+merchantability and fitness for a particular purpose, are
+disclaimed. In no event shall UCAR be liable for any damages,
+whatsoever, whether direct, indirect, consequential or special,
+that arise out of or in connection with the access, use or
+performance of WRF, including infringement actions.
+
+WRF® is a registered trademark of the University Corporation for
+Atmospheric Research (UCAR).
+
+The Weather Research and Forecasting (WRF) Model is an open source
+weather prediction system that can operate across a broad range of
+geographic scales.  We provide a series of scripts that download,
+modify configuration, build, and run the workload.
+
+The WRF build scripts use perl to change the configure file before
+build, so the generalized setup of compilers used for other apps
+in the GEOPM integration infrastructure will not work as expected.
+The configure file has been modified in the patches to use ifort,
+mpiifort, and mpiicc.  Please modify as necessary.
```

Comment:
Maybe more information about how to change the compiler?   Like "change references to ifort and mpiifort in file <FILE_PATH> to enable a different toolchain"

Reply:
Updated with path required for future/different compiler changes in the WRF model.